### DISCUSSION TITLE
Bravo

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#779] [Q&A]
### Channel Name

Bravo

### Channel ID

BravoEast.us

### Website

https://www.bravotv.com

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/bz_nexgen.m3u#L6-L7
